README.md

```diff
@@ -26,7 +26,7 @@ as [Kaniko](https://github.com/GoogleContainerTools/kaniko),
 | Dependency                                | Supported versions                     |
 | ----------------------------------------- | -------------------------------------- |
 | [Kubernetes](https://kubernetes.io/)      | v1.15.\*, v1.16.\*, v1.17.\*, v1.18.\* |
-| [Tekton](https://cloud.google.com/tekton) | v0.20.1                                |
+| [Tekton](https://cloud.google.com/tekton) | v0.21.0                                |
```

Comment:
This should link to https://tekton.dev instead of cloud.google.com/tekton

Reply:
Done